### PR TITLE
feat: add new results page UI 

### LINKED ISSUE #138: [FEATURE] Design for a user settings page.
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

We would need a page where users able to have some for of personalized features to choose from.
let's caret style, sound (whether on or off, and maybe with different sounds to choose from).
We can also even display all the themes there again.

### Proposed Solution

We need a figma design first to assist us.

### Screenshots

_No response_

### Do you want to work on this issue?

No

### If "yes" to above, please explain how you would technically implement this (issue will not be assigned if this is skipped)

_No response_

### PR DESCRIPTION
**Pull request by d-a-ve**
### What issue does this PR closes?
close #138 

### Description of file changes
- Revamped the results page to follow UI
- added a select component
- added a util function to format the time based on the time the user selects

### Is this PR still in progress or completed?
- [ ] This PR is still a work in progress (WIP)
- [ X] This PR is completed and ready for review
- [ X] Did you edit the package.json or pnpm.yaml? If yes, explain reason for modification and add the `dependencies` label to this PR. <br>
I added radix-ui/select package and also modified the eslint config.

### CODE REVIEW

File: app/(Base)/results/page.tsx [hunk 1]
```diff
@@ -1,38 +1,87 @@
...
+
+const LANGUAGES = ['English', 'Spanish', 'French', 'German'];
+const SHOW_ALL_TIME_STATS = false;
 
 export default function ResultsPage() {
    const { typedEntries, errorCount } = wordStore.getState();
    const { time, category } = userConfigStore.getState();
 
-   const rawWpm = grossWPM(typedEntries, time);
+   const rawWpmw = grossWPM(typedEntries, time);
```
Comment (JhohannesK) [inline app/(Base)/results/page.tsx:22]: why rename the variable by just adding the 'w'.. does it mean anything? 😆 😆 😆

**Review by JhohannesK — APPROVED** [review]
LGTM

File: app/(Base)/results/page.tsx (hunk 1 above)
Comment (d-a-ve) [reply app/(Base)/results/page.tsx:22]: i was the one that did that?? I honestly didn't know 🤣 . I was surprised when I saw the name changed, I just thought that was how it was and left it like that. 

Should I push a commit removing it?

Comment (JhohannesK) [reply app/(Base)/results/page.tsx:22]: I will deal with it..dw..
nice job @d-a-ve 👍🏽 
Really appreciate. 🙌🏽

**Pull request merged**